sway/commands/input/calibration_matrix.c

```diff
@@ -0,0 +1,40 @@
+#define _POSIX_C_SOURCE 200809L
+#include <string.h>
+#include <strings.h>
+#include "sway/config.h"
+#include "sway/commands.h"
+#include "sway/input/input-manager.h"
+#include "log.h"
+#include "stringop.h"
+#include "util.h"
+
+struct cmd_results *input_cmd_calibration_matrix(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "calibration_matrix", EXPECTED_EQUAL_TO, 1))) {
+		return error;
+	}
+	struct input_config *ic = config->handler_context.input_config;
+	if (!ic) {
+		return cmd_results_new(CMD_FAILURE, "No input device defined.");
+	}
+
+	list_t *split = split_string(argv[0], " ");
```

Comment:
Is there a reason why the strings need to be quoted like this:

```
calibration_matrix "1 2 3 4 5 6"
```

Instead of having something like this:

```
calibration_matrix 1 2 3 4 5 6
```

Reply:
Indeed it is simpler this way. Implemented.